recipe/meta.yaml

```diff
@@ -8,13 +8,15 @@ package:
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: 00fba178d361db5fa12787dae018c8df25207feb7eb79ec52b70259dbeab96d0
-  - url: https://static.rust-lang.org/dist/2020-04-26/rust-nightly-x86_64-unknown-linux-gnu.tar.gz
-    sha256: a723247878b5b65ec511c86108b9adb99bd1fa9b78ed87986283b93bf9e50d20
+  - url: https://static.rust-lang.org/dist/2020-06-09/rust-nightly-x86_64-unknown-linux-gnu.tar.gz  # [linux]
+    url: https://static.rust-lang.org/dist/2020-06-09/rust-nightly-x86_64-apple-darwin.tar.gz  # [osx]
```

Comment:
This line missed the leading `-`.
I've added a commit to fix it and so that we have valid YAML in any case (just to avoid any possible future issues).

Reply:
Ah, ok. I see what I was missing. Thanks for spotting it.